cp-ksqldb-cli/Dockerfile.ubi8

```diff
@@ -53,7 +53,12 @@ gpgcheck=1 \n\
 gpgkey=${CONFLUENT_PACKAGES_REPO}/archive.key \n\
 enabled=1 " > /etc/yum.repos.d/confluent.repo \
     && echo "===> Installing Confluent Hub client ..." \
-    && yum install -y confluent-hub-client-${CONFLUENT_VERSION} \
+    && yum install -y \
+        confluent-hub-client-${CONFLUENT_VERSION} \
+        # We are installing confluent-telemetry package explicitly because
+        # KSQL's deb/rpm packages cannot directly depend on
+        # confluent-telemetry package as KSQL is Open Source.
+        confluent-telemetry-${CONFLUENT_VERSION} \
```

Comment:
Maybe keep with the existing pattern and echo what you're installing before installing it